runtime/compiler/control/CompilationThread.cpp

```diff
@@ -7512,10 +7511,14 @@ TR::CompilationInfoPerThreadBase::postCompilationTasks(J9VMThread * vmThread,
    else // compilation will not be retried, either because it succeeded or because we don't want to
       {
       TR_PersistentJittedBodyInfo *bodyInfo;
+      void *extra;
```

Comment:
```suggestion
      void *extra = NULL;
``` 

Ensure this is definitely assigned